### DISCUSSION TITLE
为GitHub操作不熟练的知情人士开一个最新停贷通知书提供专楼（本项目不是烂尾楼盘收集！！！）

### DISCUSSION THREAD

**Discussion by ghost** [WeNeedHome/SummaryOfLoanSuspension#585] [General]
如题 请求管理员置顶 大量的信息散落在各处 不利于集中收集和核实
如有最新线索的知情人士 请将你们提供的信息发在本专楼 严格按照本项目的一般格式 **必须附上停贷通知书 无图的线索不予受理**
请大家持续顶贴 务必保持置顶

**Comment by yalo-ddaa** [comment]
支持楼主，我也可以帮忙一起弄PR。
提交 地址、楼盘名称和照片 即可。

**Reply by yalo-ddaa** [reply]
没用过github的同志们可以在此贴留言。希望repo主考虑一下 @WeNeedHome

**Reply by yalo-ddaa** [reply]
支持一下@WeNeedHome

**Comment by aiheihe** [comment]
热力图：https://aiheihe.github.io/WeNeedHome/
![image](https://user-images.githubusercontent.com/109200251/179191493-0557e1b4-fb5d-48f8-bdde-c6a3de31fdcc.png)

 本来昨晚提pr的 但是昨晚提交不了 
 麻烦有权限了处理一下
 不行我晚上回去了提pr

**Reply by lionwill** [reply]
改成热力图用色值区分呗，这个打点看不出来

**Comment by jiurong** [comment]
关键这个停贷通知书是否真实，是否全体业主签字同意？这个不好去甄别
